### ISSUE TITLE
Identify user groups and create user stories, and othe user information necessary for testing 

### ISSUE THREAD

**Issue by KPHowley** [hackforla/tdm-calculator#724]
### Overview
Identify all the user groups for user testing and start to create users stories 

### Action Items

- [ ] Review [User List Document](https://docs.google.com/document/d/13ct5rS3FsQQpMBQ_minqDklFLcXCWffGlZ2XePKe3c8/edit
)
- [ ] Start user story Document 
- [ ] Determine other information necessary for user testing 

### Resources/Instructions

https://docs.google.com/document/d/13ct5rS3FsQQpMBQ_minqDklFLcXCWffGlZ2XePKe3c8/edit

**Comment by abbyrivard** [comment]
User Roles as per the [Wiki](https://github.com/hackforla/tdm-calculator/wiki/User-Roles) conflicts with the current documents in this issue: need to adapt the documents to match what the wiki says.

**Comment by abbyrivard** [comment]
Need to review priority for this issue, and determine the purpose and deliverable. Is the goal to update the current google sheet? 
We currently utilize the following to gain a better sense of the user we are testing for:
- Client Provided Case Studies
- User Roles 
- Research Plans
Are these items enough to fulfill this issue's needs?

**Comment by ExperimentsInHonesty** [comment]
@abbyrivard I moved this back to the new issue approval column from the icebox because it doesn't have a dependency.

**Comment by ExperimentsInHonesty** [comment]
@mh-faraji Please provide update

<details><summary>Instructions</summary>

1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (if necessary): "Add any pictures that will help illustrate what you are working on."
</details> 

1. Progress: 
2. Blockers: 
3. Availability:
4. ETA:
5. Pictures (if necessary):

**Comment by mh-faraji** [comment]
@abbyrivard and @Melissa, I finished the [user stories](https://docs.google.com/spreadsheets/d/1xD63IrmpHGugSxOiLLZbuUUow2i0EFUDotkQ1UkBg1w/edit?pli=1&gid=1736142241#gid=1736142241) based on my understanding of the TDM Calculator website and the TDM Program Guidelines, following the visitor user story format. If you reassign me, I’m ready to start working on the Persona next.

**Comment by ExperimentsInHonesty** [comment]
@abbyrivard @melissaperry09 @mh-faraji I see some inaccurate statements in the screenshot above.  How do you want to proceed?

**Comment by ExperimentsInHonesty** [comment]
@melissaperry09 You moved this to the icebox, but it has no dependency. I moved it back to the new issue approval

**Comment by melissaperry09** [comment]
@ExperimentsInHonesty - I am moving this to questions and clarifications with a ready for product label because we are not sure how to proceed with this issue. We want to make sure we address the inaccurate statements in the above screenshot, but we are not sure which ones we need to revise. Please let me know if this would warrant time on your calendar or if we can do some asynchronous revising using comments on the spreadsheet!

**Comment by ExperimentsInHonesty** [comment]
Action item 
- [ ] product to review this [user stories](https://docs.google.com/spreadsheets/d/1xD63IrmpHGugSxOiLLZbuUUow2i0EFUDotkQ1UkBg1w/edit?pli=1&gid=1736142241#gid=1736142241) sheet, and fix or point out any inaccuracies.

**Comment by abbyrivard** [comment]
Added new column to the [User Stories](https://docs.google.com/spreadsheets/d/1xD63IrmpHGugSxOiLLZbuUUow2i0EFUDotkQ1UkBg1w/) document to capture the relevant RP that each user story is related to/pulled from

<img width="110" alt="Image" src="https://github.com/user-attachments/assets/ef4b0d45-a1ea-4b6f-b832-d5ee006f9478" />

**Comment by ExperimentsInHonesty** [comment]
@abbyrivard When you say employee do you mean city employee

**Comment by angelaliao1231** [comment]
Please provide update @abbyrivard 

<details><summary>Instructions</summary>

1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (if necessary): "Add any pictures that will help illustrate what you are working on."
</details> 

You can use this template
```
1. Progress: 
2. Blockers: 
3. Availability:
4. ETA:
5. Pictures (if necessary): 
```

**Comment by ExperimentsInHonesty** [comment]
the above comment is specifically related to making it possible for this user story to be achieved.  We need to review each story and see if it is possible to achieve it.  If not, we need to decide if we are descoping or what features might be needed.


Logged in Status | As a/an User type | I want to | so that | Acceptance Criteria
-- | -- | -- | -- | --
user-logged in | Property Manager | I need to view transportation requirements for properties I manage | so that I can ensure we maintain compliance | The property manager can access all TDM calculations for properties under their management.

**Comment by Rabia2219** [comment]
> [!NOTE]
> Not in Staging Deck

**Comment by ExperimentsInHonesty** [comment]
user stories

Property Owner / Builder Shares Initial Project Details

The building owner or developer gathers all property details (project parameters, layout, parking, etc.) and enters them into the TDM Calculator or sends the details to a consultant via email or phone.

If entered directly into the TDM Calculator, the owner can share a snapshot of the project details with their consultant.

Consultant Creates the First Project Draft

The consultant receives project details either via a shared snapshot in the TDM Calculator or from offline communication (email, phone).

If working from a shared snapshot, the consultant can make a copy in their own account to work on.

The consultant begins drafting the TDM plan using the provided details.

Consultant Produces Final TDM Plan & Shares It with Client

Once the plan is complete, the consultant shares the finalized TDM plan snapshot with the property owner, builder, or other designated recipient.

Property Owner / Builder Submits TDM Plan for Approval

The recipient of the consultant’s finalized TDM plan reviews it and submits it through the TDM Calculator for approval by the city.

Approved TDM Plan is Shared for Ongoing Compliance

After approval, the property owner or builder shares the approved TDM plan snapshot with the property management company.

This ensures the property manager has all strategies and project details needed to monitor compliance over time.

The compliance tracking itself is outside the scope of the TDM Calculator, but the plan remains accessible for reference, download, or printing.

**Comment by abbyrivard** [comment]
Per meeting with @melissaperry09 and @ExperimentsInHonesty and @abbyrivard ; This issue needs to be re-written so that Product can finish the user stories while Research can complete the column "i" section (Relevant RP to the scenario) for the user stories spreadsheet. 
Adding Ready for Prod Label so this can be restructured.

**Comment by Rabia2219** [comment]
"Approval Status" is being changed to "Submission Status" on Manage Submissions and Submissions
